code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -9,18 +9,20 @@
 	icon = 'icons/turf/floors.dmi'
 	icon_state = "asteroid"
 	icon_plating = "asteroid"
+	postdig_icon_change = TRUE
 	var/environment_type = "asteroid"
 	var/turf_type = /turf/open/floor/plating/asteroid //Because caves do whacky shit to revert to normal
-	var/dug = 0       //0 = has not yet been dug, 1 = has already been dug
-	var/sand_type = /obj/item/ore/glass
 	var/floor_variance = 20 //probability floor has a different icon state
+	archdrops = list(/obj/item/ore/glass = 5)
```

Comment:
Where is this defined? That should be the real thing calling `AddComponent()`

Reply:
It is now defined as an arg on AddComponent()